command/cmd_get.go

```diff
@@ -0,0 +1,55 @@
+package command
+
+import (
+	"fmt"
+
+	"github.com/urfave/cli/v2"
+
+	"github.com/peak/s5cmd/storage"
+)
+
+var GetCommand = &cli.Command{
+	Name:     "get",
+	HelpName: "get",
+	Usage:    "TODO",
```

Comment:
Missin usage

Reply:
`get` command is gone. Please see the description.